src/Stratis.Bitcoin.IntegrationTests/Mempool/MemoryPoolTests.cs

```diff
@@ -127,70 +127,6 @@ public void MempoolReceiveFromManyNodes()
             }
         }
 
-        [Fact]
-        [Trait("Unstable", "True")]
-        public void TxMempoolBlockDoublespend()
```

Comment:
Do we have any other tests that checks double spend?

Reply:
Don't know, most likely no.

Reply:
@noescape00 @dangershony I believe `sending_transaction_with_double_spend_in_mempool` and `sending_transaction_with_double_spend_mined_trx` covers double spends.